.github/workflows/ci.yml

```diff
@@ -26,6 +26,6 @@ jobs:
         run: |
           npm install
           npm run lint
-          npx pretty-quick --check .
```

Comment:
is it not working well? or you want to format all files regardless if they're changed?

Reply:
@boudra I added `npm run prettier-check` which is defined as the same here so that we can call it directly in local  with the same internal command

Reply:
yeah it makes sense i don't know why I read it as `npx prettier check`